impeller/renderer/backend/vulkan/pipeline_cache_vk.cc

```diff
@@ -68,13 +68,6 @@ PipelineCacheVK::PipelineCacheVK(std::shared_ptr<const Capabilities> caps,
       OpenCacheFile(cache_directory_, kPipelineCacheFileName, vk_caps);
 
   vk::PipelineCacheCreateInfo cache_info;
-
-  // TODO(csg): VK_PIPELINE_CACHE_CREATE_EXTERNALLY_SYNCHRONIZED_BIT is behind
-  // an extension. Check it and set it. If not, the implementation is doing
-  // unnecessary synchronization.
-  // cache_info.flags =
-  // vk::PipelineCacheCreateFlagBits::eExternallySynchronized;
```

Comment:
Based on what I'm reading in https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VkPipelineCacheCreateFlagBits.html it sounds like the idea would be to use this extension so we could use the mutex ourselves.

But since we don't have the extension it ends up being both us and the implementation blocking.

So we might as well just let the implementation do it for us.